ipfshttpclient/client/dag.py

```diff
@@ -69,6 +70,8 @@ def put(self, data: utils.clean_file_t, **kwargs: base.CommonArgs):
 			dict
```

Comment:
The parameter section here should document the `format` and `input_enc` parameters

Reply:
Done :)